.github/workflows/autobuild.yml

```diff
@@ -65,92 +73,190 @@ jobs:
+        
+           - config_name:           Linux (artifacts)
              target_os:             linux
-             building_on_os:        ubuntu-18.04
-             asset_path:            deploy/Jamulus_headless_amd64.deb
-             asset_name:            jamulus_headless_${{ needs.create_release.outputs.version_name }}_ubuntu_amd64.deb
-           - config_name:           MacOS
+             building_on_os:        ubuntu-20.04
+             cmd1_prebuild:         "sh linux/autobuild/artifacts/autobuild_deb_1_prepare.sh "
+             cmd2_build:            "sh linux/autobuild/artifacts/autobuild_deb_2_build.sh "
+             cmd3_postbuild:        "sh linux/autobuild/artifacts/autobuild_deb_3_copy_files.sh "
+             uses_codeql:           false
+             
+           - config_name:           MacOS (codeQL)
+             target_os:             macos
+             building_on_os:        macos-10.15
+             cmd1_prebuild:         "sh mac/autobuild/codeQL/autobuild_mac_1_prepare.sh "
+             cmd2_build:            "sh mac/autobuild/codeQL/autobuild_mac_2_build.sh "
+             cmd3_postbuild:        "sh mac/autobuild/codeQL/autobuild_mac_3_copy_files.sh "
+             uses_codeql:           true
+             
+           - config_name:           MacOS (artifacts)
              target_os:             macos
              building_on_os:        macos-10.15
-             asset_path:            deploy/Jamulus-installer-mac.dmg
-             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_mac.dmg
-           - config_name:           Windows
+             cmd1_prebuild:         "sh mac/autobuild/artifacts/autobuild_mac_1_prepare.sh "
+             cmd2_build:            "sh mac/autobuild/artifacts/autobuild_mac_2_build.sh "
+             cmd3_postbuild:        "sh mac/autobuild/artifacts/autobuild_mac_3_copy_files.sh "
+             uses_codeql:           false
+             
+           - config_name:           Windows (artifact+codeql)
              target_os:             windows
              building_on_os:        windows-latest
-             asset_path:            deploy\Jamulus-installer-win.exe
-             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_win.exe
-           - config_name:           AndroidAPK
-             target_os:             android
-             building_on_os:        ubuntu-18.04
-             asset_path:            android-build/build/outputs/apk/debug/android-build-debug.apk
-             asset_name:            jamulus_${{ needs.create_release.outputs.version_name }}_android.apk
+             cmd1_prebuild:         powershell ./windows/autobuild/autobuild_windowsinstaller_1_prepare.ps1
+             cmd2_build:            powershell ./windows/autobuild/autobuild_windowsinstaller_2_build.ps1
+             cmd3_postbuild:        powershell ./windows/autobuild/autobuild_windowsinstaller_3_copy_files.ps1
```

Comment:
I assume my PR (with the nsProcess building) is included

Reply:
yes, merged as 81ebb786c34c70d0c5ac78e39bcfca48acb4ed4a

Reply:
Yes. Saw that too.